server/common.js

```diff
@@ -19,10 +19,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 const questionTypes = Object.freeze({
-    MULTIPLECHOICE  : {name: 'Multiple Choice', value: 'mc', template: 'mc-answer', icon: 'format_list_bulleted'},
```

Comment:
what happened here? why do we no longer use the icons?

Reply:
whoops,  I think when merging it I might have removed that. Let me put it back.